### PR TITLE
MAINT: removed data parser class

### PR DESCRIPTION
**Pull request by rpartzsch**
Removed the data parser class, interpreted output path can be set as None. Also ASCII art in `aidat_tlu_run` is now a raw string.

### CODE REVIEW

File: aidatlu/main/tlu.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 from aidatlu.hardware.ioexpander_controller import IOControl
 from aidatlu.hardware.trigger_controller import TriggerLogic
 from aidatlu.main.config_parser import TLUConfigure
-from aidatlu.main.data_parser import DataParser
+import aidatlu.main.data_parser as DataParser
```
Comment (cbespin) [inline aidatlu/main/tlu.py:20]: While this works, why import the whole module? Can you not import `interpret_data()`? If you need `_create_table()`, just move it inside `interpret_data()`. Keeps the code cleaner than using a module like a class.

File: aidatlu/main/tlu.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def __init__(self, hw, config_path, clock_config_path) -> None:
 
         self.reset_configuration()
         self.config_parser = TLUConfigure(self, self.io_controller, config_path)
-        self.data_parser = DataParser()
+        self.data_parser = DataParser
```
Comment (cbespin) [inline aidatlu/main/tlu.py:44]: Related to above

File: aidatlu/test/software_test.py
Comment (cbespin) [inline aidatlu/test/software_test.py:1]: Related to above

**Review by cbespin — CHANGES_REQUESTED** [review]

File: aidatlu/main/tlu.py (hunk 1 above)
Comment (rpartzsch) [reply aidatlu/main/tlu.py]: Yes, I dont even need `_create_table()`.

**Review by cbespin — APPROVED** [review]

**Pull request merged**